packages/frontend/src/components/DashboardTiles/DashboardChartTile.tsx

```diff
@@ -171,24 +171,38 @@ const DashboardChartTile: FC<Props> = (props) => {
             const dimensions = getDimensions(explore).filter((dimension) =>
                 e.dimensionNames.includes(fieldId(dimension)),
             );
-            setDashboardFilterOptions(
-                dimensions.map((dimension) => ({
-                    id: uuidv4(),
-                    target: {
-                        fieldId: fieldId(dimension),
-                        tableName: dimension.table,
-                    },
-                    operator: FilterOperator.EQUALS,
-                    values: [e.data[fieldId(dimension)]],
-                })),
-            );
+
+            const dimensionOptions = dimensions.map((dimension) => ({
+                id: uuidv4(),
+                target: {
+                    fieldId: fieldId(dimension),
+                    tableName: dimension.table,
+                },
+                operator: FilterOperator.EQUALS,
+                values: [e.data[fieldId(dimension)]],
+            }));
+            const pivot = savedQuery?.pivotConfig?.columns?.[0];
+            const pivotOptions = pivot
+                ? [
+                      {
+                          id: uuidv4(),
+                          target: {
+                              fieldId: pivot,
+                              tableName: pivot.split('_')[0],
+                          },
+                          operator: FilterOperator.EQUALS,
+                          values: [e.seriesName],
```

Comment:
I don't think this works since the series name is a hash generated with this function 
`hashFieldReference`

Reply:
How about now ? I didn't find anything specific about pivot values on the series 

![Peek 2022-07-12 15-54](https://user-images.githubusercontent.com/1983672/178507949-aee1c54c-529d-4ea4-9b15-74a18e245715.gif)


![image](https://user-images.githubusercontent.com/1983672/178507878-27d1d474-9774-4479-a3ad-756fc03dcb36.png)